modules/ROOT/pages/auth.adoc

```diff
@@ -0,0 +1,170 @@
+= Luna Streaming Authentication
+
+== Authentication
```

Comment:
Move headings up a level b/c everything is in authentication, which is this entire topic.